pypeit/core/save.py

```diff
@@ -646,56 +544,59 @@ def save_obj_info(all_specobjs, fitstbl, spectrograph, basename, science_dir):
 
 
 def save_2d_images(sci_output, fitstbl, scidx, ext0, setup, mfdir,
```

Comment:
Have you tested the pypeit_show_2dspec script on this?

Reply:
Tested on recently reduced GMOS data.
Worked fine.